interface/billing/sl_eob_search.php

```diff
@@ -722,9 +722,9 @@ function toEncSummary(pid) {
 </head>
 
```

Comment:
This page has a few issues here circled:
![image](https://user-images.githubusercontent.com/46497068/92337098-2e7ceb80-f075-11ea-9f5a-f3814f393174.png)